apps/angular/src/app/DTOs/AiredDTO.ts

```diff
@@ -0,0 +1,9 @@
+/** Aired type. */
+export type AiredDTO = {
+
+	/** Start. */
+	start: string;
+
+	/** End. */
+	end: string | null;
+};
```

Comment:
Pls, revise code for adding missed `readonly` modifiers.
```suggestion
export type AiredDTO = {

	/** Start. */
	readonly start: string;

	/** End. */
	readonly end: string | null;
};
```